app/modules/sightings/parameters.py

```diff
@@ -18,6 +17,21 @@ class Meta(schemas.DetailedSightingSchema.Meta):
 
 class PatchSightingDetailsParameters(PatchJSONParameters):
     # pylint: disable=abstract-method,missing-docstring
-    OPERATION_CHOICES = (PatchJSONParameters.OP_REPLACE,)
+    OPERATION_CHOICES = (
+        PatchJSONParameters.OP_ADD,
+        PatchJSONParameters.OP_REPLACE,
+        PatchJSONParameters.OP_REMOVE,
+    )
 
-    PATH_CHOICES = tuple('/%s' % field for field in (Sighting.guid.key,))
+    PATH_CHOICES_EDM = (
```

Comment:
Why not make this PATH_CHOICES and ditch the PATH_CHOICES_EDM?

Reply:
in the event we need to support _both_ patchable properties *on the houston object* and on *edm* objects; plus needing to distinguish which ones are to be passed through to edm.   there is not a lot of properties on houston-Sighting right now, but say for the sake of argument we wanted to patch path=/version (we dont, but..) .. "/version" would live in PATH_CHOICES while edm-specific ones live in PATH_CHOICES_EDM  ... which is used in `sightings/resources.py` to conditionally branch to the "do the patch in EDM not houston" code:  `.... in parameters.PatchSightingDetailsParameters.PATH_CHOICES_EDM .... `

but _both_ types needs to first be seen as legit paths by whatever patch magic happens inside flask, hence these edm values residing in PATH_CHOICES as well.   so i am just letting PATH_CHOICES be the union of houston-properties and edm-properties.  it just so happens the first set is empty!